app/components/report-filter/template.hbs

```diff
@@ -0,0 +1,102 @@
+<div class="grid grid--2of12">
+  {{#task-selection
```

Comment:
Don't we also need to check here if `customer`, `project` or `task` are enabled?